### ISSUE TITLE
_index/_prevIndex trigger error TS7031 Binding elements implicitly have an 'any' type.

### ISSUE THREAD

**Issue by david-alza** [gusgard/react-native-swiper-flatlist#139]
React Native version: 0.68.2 (within Expo)
## Steps To Reproduce
We hit this in our repo when upgrading from Expo 44 -> Expo 45 while also bumping `@types/react` up to `18.0.12` which includes this change: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/52873

This isn't a big deal, nor is it truly a bug, but I thought I'd document it here as a reference for an incoming pull request. 

```
node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx:79:17 - error TS7031: Binding element '_index' implicitly has an 'any' type.

79       ({ index: _index, prevIndex: _prevIndex }) => {
                   ~~~~~~

node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx:79:36 - error TS7031: Binding element '_prevIndex' implicitly has an 'any' type.

79       ({ index: _index, prevIndex: _prevIndex }) => {
                                      ~~~~~~~~~~


Found 2 errors in the same file, starting at: node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx:79
```

Specifically if those arguments get typed as numbers then the error goes away: https://github.com/gusgard/react-native-swiper-flatlist/blob/master/src/components/SwiperFlatList/SwiperFlatList.tsx#L79

**Comment by Linyan-ma** [comment]
will this be fixed?

**Comment by gusgard** [comment]
Thanks @david-alza for the fix, it was released in `3.0.17`

**Comment by gusgard** [comment]
@Linyan-ma ☝️
